terraform/modules/emr/templates/emr/hdfs_setup.sh

```diff
@@ -39,6 +39,7 @@ set -o pipefail
   #### Fix up Tez
   sudo -H -u hdfs bash -c "hdfs dfs -mkdir -p /libs"
   sudo -H -u hdfs bash -c "hdfs dfs -put /usr/lib/hbase/hbase-client*.jar /libs"
+  sudo -H -u hdfs bash -c "hdfs dfs -put /usr/share/java/Hive-JSON-Serde/hive-openx-serde.jar /libs"
```

Comment:
Are you sure this library exists?

Reply:
i.e. have you tried this on the analytical-env cluster in dev before submitting a PR?

Reply:
Tested and SerDe now working